python/cac_tripplanner/destinations/models.py

```diff
@@ -37,9 +37,9 @@ class Meta:
                                           'but may be overridden manually if incorrect.'))
     point = models.PointField()
     image = models.ImageField(upload_to=generate_filename, null=True,
-                              help_text='The full-size image. Will be displayed at 400x400.')
+                              help_text='The full-size image. Will be displayed at 310x155.')
     wide_image = models.ImageField(upload_to=generate_filename, null=True,
-                                   help_text='The half-height image. Will be displayed at 400x200.')
+                                   help_text='The half-height image. Will be displayed at 680x400.')
```

Comment:
The "Full-size" / "Half-height" and "Narrow" / "Wide" terms no longer really apply to the new sizes.

Would it be possible to change the display names of the image fields in the admin UI on both Articles and Destinations to simply: "Large image" and "Small image"?

Regardless of whether we can do that, let's change the help text, for both articles and destinations, to:

"The large image.  Will be displayed at {w}x{h}."
"The small image. Will be displayed at {w}x{h}."